app/views/pages/_editor_guide_text.html.erb

```diff
@@ -80,14 +80,34 @@
+    <pre># One '#' for a h1 heading<br>## Two '#'s for a h2 heading<br>...<br>###### Six '#'s for a h6 heading</pre>
 
     <%= render partial: "editor_guide_h3", locals: { id: "author-notes", title: "Author Notes/Comments" } %>
     <p>Add some hidden notes/comments to your article with this syntax:</p>
     <pre>&lt;!-- This won't show up in the content! --&gt;</pre>
 
+    <h2 style="font-size:2.8em" id="accessibility" class="permalink-heading">
+      <strong>Accessibility</strong>
+      <a href="#accessibility" aria-label="permalink for Accessibility"><%= inline_svg_tag("small-link.svg", aria_hidden: true) %></a>
+    </h2>
+    <p>People access online content in all kinds of different ways, and there are a few things you can do to make your posts more easily understood by a broad range of users. You can find out more about web accessibility at <a href="https://www.w3.org/WAI/fundamentals/accessibility-intro/">W3C's Introduction to Web Accessibility</a>, but there are two main ways you can make your posts more accessible: providing alternative descriptions of any images you use, and adding appropriate headings.</p>
+
+    <%= render partial: "editor_guide_h3", locals: { id: "alt-text-for-images", title: "Providing alternative descriptions for images" } %>
+    <p>Some users might not be able to see or easily process images that you use in your posts. Providing an alternative description for an image helps make sure that everyone can understand your post, whether they can see the image or not.</p>
+    <p>When you upload an image in the editor, you will see the following text to copy and paste into your post:</p>
+    <pre>![Alt Text](/file-path/my-image.png)</pre>
+    <p>Replace the "Alt Text" in square brackets with a description of your image - for example:</p>
+    <pre>![A pie chart showing 40% responded "Yes", 50% responded "No" and 10% responded "Not sure"](/file-path/my-image.png)</pre>
+    <p>By doing this, if someone reads your post using an assistive device like a screen reader (which turns written content into spoken audio) they will hear the description you entered.</p>
+
+    <%= render partial: "editor_guide_h3", locals: { id: "accessible-headings", title: "Providing headings" } %>
+    <p>Headings provide an outline of your post to readers, including people who can't see the screen well. Many assistive technologies (like screen readers) allow users to skip directly to a particular heading, helping them find and understand the content of your post with ease.</p>
+    <p><a href="#markdown-headers">Headings can be added in levels 1 - 6</a>. Avoid using a level one heading (i.e. '# Heading text') - when you create a post, your post title automatically becomes a level one heading and serves a special role on the page, much like the headline of a newspaper article. Similar to how a newspaper article only has one headline, it can be confusing if multiple level one headings exist on a page.</p>
```

Comment:
```suggestion
    <p><a href="#markdown-headers">Headings can be added in levels 1 - 6</a>. Avoid using a level one heading (i.e., '# Heading text'). When you create a post, your post title automatically becomes a level one heading and serves a special role on the page, much like the headline of a newspaper article. Similar to how a newspaper article only has one headline, it can be confusing if multiple level one headings exist on a page.</p>
```